plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerDistributedQueries.java

```diff
@@ -103,6 +103,11 @@ protected TestTable createTableWithDefaultColumns()
             return Optional.empty();
         }
 
+        if (typeName.contains("char(1)")) {
```

Comment:
Why is `varchar` not skipped here?

Reply:
Oh - I did not notice that. Please change to `equals || equals` :)

Reply:
Or - actually - NVM - you get rid of that anway.